application/src/test/java/org/togetherjava/tjbot/commands/utils/MessageUtilsTest.java

```diff
@@ -188,4 +194,61 @@ void abbreviate() {
     private record TestCaseAbbreviate(String testName, String abbreviatedMessage,
             String originalMessage, int limit) {
     }
+
+    private static Stream<Arguments> provideExtractCodeTests() {
+        return Stream.of(createExtractCodeArgumentsFor("basic", """
```

Comment:
All honesty unreadable, honestly, using a list and using separate List#add calls would be a lot more readable

Reply:
i doubt it would be more readable after spotless. but sure, can give it a try

Reply:
done ✔️ 

i think its actually more readable now, thanks